elixir/raindrops/lib/raindrops.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Raindrops do
+  @doc """
+  Returns a string based on raindrop factors.
+
+  - If the number contains 3 as a prime factor, output 'Pling'.
+  - If the number contains 5 as a prime factor, output 'Plang'.
+  - If the number contains 7 as a prime factor, output 'Plong'.
+  - If the number does not contain 3, 5, or 7 as a prime factor,
+    just pass the number's digits straight through.
+  """
+
+  @prime_factors [3, 5, 7, 1]
+  @sounds %{3 => "Pling", 5 => "Plang", 7 => "Plong", 1 => ""}
```

Comment:
I love it when you extract data from functions. Just cleans up everything :heart_eyes: